roqoqo/src/operations/pragma_operations.rs

```diff
@@ -1059,6 +1052,12 @@ pub struct PragmaControlledCircuit {
 
 impl super::ImplementedIn1point5 for PragmaControlledCircuit {}
 
+impl SupportedVersion for PragmaControlledCircuit {
+    fn minimum_supported_roqoqo_version(&self) -> (u32, u32, u32) {
+        (1, 5, 0)
+    }
+}
+
```

Comment:
We should also check the internal circuit and take the maximum of 1.5 and the minimum version of the internal circuit as the overall minimal version.